libs/template/presets/b2b-storefront/app.ts

```diff
@@ -1,9 +1,9 @@
 import { AppFeature } from '@spryker-oryx/core';
 import { provideExperienceData } from '@spryker-oryx/experience';
 import { storefrontFeatures } from '@spryker-oryx/presets/storefront';
-import { cartsPage } from './experience/carts-page';
 import { contactPage } from './experience/contact-page';
 import { HeaderTemplate } from './experience/header';
+import { cartsPage } from './experience/pages';
```

Comment:
Wonder if it should be part of generic pages as from oryx pov multi-cart is not b2b only